packages/ra-ui-materialui/src/input/AutocompleteInput.tsx

```diff
@@ -345,7 +345,7 @@ If you provided a React element for the optionText prop, you must also provide t
                 return;
             }
 
-            setFilters(filterToQuery(filter), undefined, true);
```

Comment:
This didn't make sense as the call is debounced two lines earlier. The previous code caused a double debounce.